explainer.md

```diff
@@ -499,6 +512,8 @@ interface VRDevice : EventTarget {
 
   Promise<boolean> supportsSession(VRSessionCreateParametersInit parameters);
   Promise<VRSession> requestSession(VRSessionCreateParametersInit parameters);
+
+  Promise<void> ensureContextCompatibility(WebGLRenderingContext context);
```

Comment:
We should use `WebGLRenderingContextBase` instead of `WebGLRenderingContext` so that the API can be used with WebGL 2.